junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -69,4 +69,11 @@
 	 */
 	char delimiter() default ',';
 
+	/**
+	 * If the CSV file resources have headers in their first line; This will activate the behaviour to skip the first
+	 * line lines for test purposes.
+	 *
+	 * <p>Defaults to {@code "false"}</p>
+	 */
+	boolean hasHeaders() default false;
```

Comment:
Thinking about letting the user specify the amount of initial lines to skip, defaults to `0`.

Reply:
Do CSV files usually have multiple lines of headers? Anyway, it would certainly be more flexible and not harder to configure, e.g. `skipHeaderLines = 1`.

Reply:
Sure! I could refactor this after work.